sdk/nodejs/appservice/zMixins.ts

```diff
@@ -525,19 +565,14 @@ export class CallbackFunctionApp<C extends Context<R>, E, R extends Result> exte
      */
     public readonly endpoint: pulumi.Output<string>;
 
-    constructor(name: string, bindings: pulumi.Input<BindingDefinition[]>,
+    constructor(name: string, bindings: pulumi.Input<BindingDefinition[]> | Function,

```

Comment:
consider renamin to bindingsOrFunction

Reply:
isn't this a breaking change?

Reply:
nope.  names of parameters aren't aavailable in any meaningful form in js.  you can also write the above as:

```ts
constructor(name: string, bindings: pulumi.Input<BindingDefinition[]>);
constructor(name: string, func: Function);
constructor(name: string, bindingsOrFunc: pulumi.Input<BindingDefinition[]> | Function) { }
```

Though that's really up to you as the union type form is also ok.